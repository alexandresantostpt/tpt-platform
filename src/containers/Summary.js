import PropTypes from 'prop-types'
import React from 'react'
import styled from 'styled-components'

const StyledSummary = styled.section`
    background: #fbfbfb;
    border: 1px solid #dcdcdc;
    border-right: none;
    grid-area: summary;
    overflow-y: auto;
    padding: 3.125rem 1.1875rem;
`

const Summary = ({ children }) => <StyledSummary>{children}</StyledSummary>

Summary.defaultProps = {
    children: null
}

Summary.propTypes = {
    children: PropTypes.oneOfType([PropTypes.array, PropTypes.object])
}

export default Summary
