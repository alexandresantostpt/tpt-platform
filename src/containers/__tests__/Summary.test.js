import React from 'react'
import renderer from 'react-test-renderer'

import Summary from '../Summary'

describe('Tests for Summary container', () => {
    it('Should rendering without crash ', () => {
        const tree = renderer
            .create(
                <Summary>
                    <h1>Must be show</h1>
                </Summary>
            )
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})
