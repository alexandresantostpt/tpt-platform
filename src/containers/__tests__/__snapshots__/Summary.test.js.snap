// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for Summary container Should rendering without crash  1`] = `
<section
  className="Summary__StyledSummary-sc-1mglm3x-0 fKeRKj"
>
  <h1>
    Must be show
  </h1>
</section>
`;
